internal/config/defs.go

```diff
@@ -1,8 +1,19 @@
 package config
 
-import "time"
+import (
+	"os"
+	"time"
+)
 
 const (
 	// DefaultTimeout is the default TURN credential timeout. Default is one day, in seconds.
 	DefaultTimeout = 24 * time.Hour
 )
+
+var PublicIp string
+
+func init() {
+	envPublicIp, present := os.LookupEnv("STUNNER_PUBLIC_IP"); if present {
+		PublicIp = envPublicIp
+	}
```

Comment:
First, I'd rather move this into `main.go`, that is the right place to process defaults. We try to avoid using `init()` as much as we can, since it secretly adds behavior whatever is importing this code. It's better to be explicit about these things.  

Second, it's maybe more idiomatic to write:
```
if envPublicIp, present := os.LookupEnv("STUNNER_PUBLIC_IP"); present {
	PublicIp = envPublicIp
}
```

Reply:
Both done, thanks!